### ISSUE TITLE
[Bug]: Whisky is damaged and can't be opened

### ISSUE THREAD

**Issue by efossvold** [IsaacMarovitz/Whisky#58]
### Description

After downloading Whisky (pre-0.2.2) and trying to open it (MacOS 13.4) I'm getting the error "Whisky is damaged and can't be opened. You should move it to the Bin".

Things I tried to resolve the issue:
* sudo spctl –master-disable (didn't work)
* sudo xattr -cr "~/Downloads/Whisky.app" (didn't work)
* cp -xr Whisky.app Whisky2.app (worked)

The last command worked and I could launch the copied app (Whisky2). Not sure if there's some permission issues with the last build (pre-0.2.2) or if I'm the only one encountering this problem.

### Steps to reproduce

1. Download Whisky (pre-0.2.2)
2. Double click on Whisky to open it

### Expected behaviour

Whisky app should open without any issues

### What version of Whisky are you using?

pre-0.2.1

### What version of macOS are you using?

Ventura (macOS 13)

### Issue Language

- [X] Yes my issue is written in English

**Comment by efossvold** [comment]
Under **What version of Whisky are you using?** it says pre-0.2.1 as there's no pre-0.2.2 option yet.

**Comment by jipiboily** [comment]
Very likely a duplicate of #57 (that I created a few minutes before, probably writing at the same time ;))

I'll let the maintainer(s) see which one to keep, or not, or to add to. Yours at least seems to have a solution, which I"ll try just now!

**Comment by jipiboily** [comment]
Just FYI, `cp -xr Whisky.app Whisky2.app` didn't work for me. Still getting the same error.

**Comment by efossvold** [comment]
> Very likely a duplicate of #57 (that I created a few minutes before, probably writing at the same time ;))
> 
> I'll let the maintainer(s) see which one to keep, or not, or to add to. Yours at least seems to have a solution, which I"ll try just now!

Yes, funny, looks like we were writing at the same time :) I checked for this issue before writing. I guess we can just let the maintainer(s) decide which one to close/keep as you said.

**Comment by efossvold** [comment]
> * sudo spctl –master-disable

Might be that you also need to run
`sudo spctl --master-disable`

However, so far I'm also blocked by #56 on my setup which seems to be Ventura related.

**Comment by jipiboily** [comment]
Ah, could have been! Doesn't look like it; just tried now. Thanks anyways!

**Comment by chadlavi** [comment]
There was something in the Readme about this that they removed for some reason, you can see it here: https://github.com/IsaacMarovitz/Whisky/commit/f9ca4b9e333e201accb0852687e255ae4f86b905#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R52

**Comment by Marsgames** [comment]
I think the macOS 13.x support is not yet deployed, so you would have to build it yourself from source or wait for the next release

**Comment by IsaacMarovitz** [comment]
Should be fixed in pre-0.2.2

**Comment by efossvold** [comment]
> Should be fixed in pre-0.2.2

As I wrote in the issue description this was with pre-0.2.2, so I don't believe it's fixed. I see that the pre-0.2.2 version was re-released some 30 mins ago. I tried it and the issue still persist. Please open the ticket again.

**Comment by efossvold** [comment]
> There was something in the Readme about this that they removed for some reason, you can see it here: [f9ca4b9#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R52](https://github.com/IsaacMarovitz/Whisky/commit/f9ca4b9e333e201accb0852687e255ae4f86b905#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R52)

So running `xattr -d com.apple.quarantine path-to-whisky` makes it work (with the pre-0.2.2 re-release)
